### PR TITLE
Run unit tests in CI

### PR DESCRIPTION
**Pull request by mohsen1**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,4 +13,15 @@ jobs:
...
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/setup-node@v3
+          with:
+            node-version: 16
+        - uses: actions/checkout@v2
+        - name: Install modules
+          uses: bahmutov/npm-install@v1
+        - name: Run unit tests
+          run: yarn test
```
Comment (grzpab) [inline .github/workflows/ci.yml:27]: can you ensure a time limit on the server (let's say 20s)?

Comment (mohsen1) [reply .github/workflows/ci.yml:27]: the smallest is 1 minute. I don't think it's necessary as it's running super fast right now

Comment (gregintuitaio) [reply .github/workflows/ci.yml:27]: do you think it makes sense to add it so we never run out of credits?

Comment (mohsen1) [reply .github/workflows/ci.yml:27]: for open source it's all free.

**Pull request merged**